lang/cs/Org.Apache.REEF.Common/Telemetry/DefaultMetricsSink.cs

```diff
@@ -39,11 +39,11 @@ private DefaultMetricsSink()
         /// Simple sink for metrics data
         /// </summary>
         /// <param name="metrics">A collection of metrics data in Key value pair format.</param>
-        public void Sink(IEnumerable<KeyValuePair<string, string>> metrics)
+        public void Sink(IEnumerable<KeyValuePair<string, IMetric>> metrics)
```

Comment:
Because you are sending the whole IMetric, there is no reason to use a KeyValuePair. Just use IEnumerable<IMetric>